packages/components/src/components/Flyout/styles/FlyoutStyles.ts

```diff
@@ -0,0 +1,39 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { sv } from '#/utilities/styleUtilities';
+
+export const flyoutContentStyles = sv({
+    base:
+        'tw-block tw-bg-base tw-shadow-mid tw-border tw-border-line tw-w-full sm:tw-w-fit sm:tw-max-w-[--flyout-max-width] tw-group tw-mt-4 sm:tw-mt-0 ' +
+        // using a special selector to set the transform of the wrapper component in the mobile view to allow for a full width flyout
+        '[body>[data-radix-popper-content-wrapper]:has(&)]:max-sm:tw-p-2 [body>[data-radix-popper-content-wrapper]:has(&)]:max-sm:tw-w-screen [body>[data-radix-popper-content-wrapper]:has(&)]:max-sm:!tw-translate-x-0 [body>[data-radix-popper-content-wrapper]:has(&)]:max-sm:!tw-translate-y-0 ' +
+        // add a full page pseudo element to darken the background
```

Comment:
```suggestion
        // Mobile view dialog background
```

Reply:
not correct, not only on mobile, but adjusted the label

Reply:
fixed